_saas-integrations/kustomer/v1/kustomer-v1.md

```diff
@@ -0,0 +1,133 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/integration-templates/saas/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+
+# -------------------------- #
+#      Page & Formatting     #
+# -------------------------- #
+
+title: Kustomer
+permalink: /integrations/saas/kustomer
+keywords: kustomer, integration, schema, etl kustomer, kustomer etl, kustomer schema
+layout: singer
+# input: false
+
+key: "kustomer-setup"
+
+
+# -------------------------- #
+#         Tap Details        #
+# -------------------------- #
+
+name: "Kustomer"
```

Comment:
This should be `kustomer`. This is also why the schemas aren't showing up on the rendered page - the value in the schema files is `kustomer`, while this is `Kustomer`. By default, when Liquid checks for equal values, they're case sensitive, meaning that the values must match *exactly*, and a difference in case (`k` vs `K`) will make the values not equal.